requirements.txt

```diff
@@ -4,6 +4,7 @@ boto==2.48.0
 boto3==1.6.8
 botocore==1.9.8
 coverage==4.5.1
+paramiko==2.6.0
```

Comment:
currently this is already `2.6.0` locally for me, I guess this forces upgrades for other installation?

Reply:
would be wary of major versions but anything `2.6.*` should be pretty much ok

Reply:
hmm, not sure. let me destroy my venv and re-install ... yeah, there it is. I'll close this PR then